packages/grafana-ui/src/components/BigValue/BigValueLayout.tsx

```diff
@@ -398,7 +405,9 @@ export class StackedWithNoChartLayout extends BigValueLayout {
         this.valueToAlignTo,
         this.maxTextWidth,
         this.maxTextHeight - titleHeight,
-        LINE_HEIGHT
+        LINE_HEIGHT,
+        undefined,
```

Comment:
Is `maxSize` commonly used? I assume that we don't want to change the order of the parameters

Reply:
I think they're used the exact same amount of times, so probably leave it like this? 🤷